can/base-map/base-map_test.js

```diff
@@ -38,26 +39,35 @@ QUnit.test("uses idProp", function(assert) {
 });
 
 
-QUnit.skip("creates map if none is provided (#8)", function(){
-
-	var connection = baseMap({
-		url: "/api/restaurants",
-		idProp: '_id',
-		name: "restaurant"
-	});
-
-	fixture({
-		"GET /api/restaurants/{_id}": function(request){
-			return {id: 5};
-		}
-	});
-
-	var done = assert.async();
-	connection.getData({_id: 5}).then(function(data){
-		assert.deepEqual(data, {id: 5}, "findOne");
-		done();
-	});
+canTestHelpers.devOnlyTest("throws an error if neither Map nor ObjectType are provided", function(assert) {
+	assert.throws(
+		function() {
+			baseMap({
+				idProp: '_id',
+				List: function() {},
+				name: "restaurant",
+				url: "/api/restaurants"
+			});
+		},
+		/must be configured with a Map or ObjectType type/g,
+		"throws"
+	);
+});
 
+canTestHelpers.devOnlyTest("throws an error when List is accessed if neither List nor ArrayType are provided", function(assert) {
+	assert.throws(
+		function() {
+			var connection = baseMap({
+				idProp: '_id',
+				Map: function() {},
+				name: "restaurant",
+				url: "/api/restaurants"
+			});
+			connection.List;
```

Comment:
I'm surprised jshint doesn't complain about this. Maybe it's not configured right?